sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/AmqpTransaction.java

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.amqp;
+
+import com.azure.core.amqp.implementation.TransactionCoordinator;
+
+import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
+import java.util.Objects;
+
+/**
+ * Represents transaction in message broker. {@link TransactionCoordinator} uses this to perform many message broker
```

Comment:
Transaction coordinator is an implementation detail. It shouldn't be mentioned in the public API.